### PR TITLE
Money Rework

### PR DESCRIPTION
**Pull request by carlarctg**
# About the pull request

Added a variable to paygrades called pay_multiplier. This multiplies the starting amount of money from bank accounts.

Refactored how bank accounts are created so the above could work.

Drastically nuked the amount of money people start with. People can no longer start with thousands of dollars.... they now get 30-50. This value is multiplied by the pay_multiplier below.

Added pay_multiplier to all paygrades. The higher your rank, the more money you'll start with, based on this multiplier. (For example, a Major will have a pay multiplier of 4.) Includes strange roles like VAIPO, UPP, PMCs, RESS...

Non-binary WY executives may now spawn with 'Mx.' as their communications prefix.

Altered the prices of cigarette vending machines around to overall make them more expensive. PFCs will not be able to buy Executive Select with their starting cash.

Made cassetes and Souto from vendors more expensive. Buying food from Hot Foods now costs money. Marine coffee now has an appropiate description. Souto vendors no longer vend water bottles.

Fixed default parent type dollar items being worth 0 money...

# Explain why it's good for the game

> Drastically nuked the amount of money people start with. People can no longer start with thousands of dollars.... they now get 30-50. This value is multiplied by the pay_multiplier below.

Lore. The live of a private sucks. Monkeysfist suggested this value. Still enough to buy all essentials, scavenge some money if you want to buy the good cigarette packs down in the colony.

> Added pay_multiplier to all paygrades. The higher your rank, the more money you'll start with, based on this multiplier. (For example, a Major will have a pay multiplier of 4.) Includes strange roles like VAIPO, UPP, PMCs, RESS...

Why were paygrades added without affecting pay. Why could PFCs start with 3 thousand dollars and COs with 50 dollars total.

> Non-binary WY executives may now spawn with 'Mx.' as their communications prefix.

Inclusivity win! Doesn't actually do anything as we do not have nonbinary characters.

> Altered the prices of cigarette vending machines around to overall make them more expensive. PFCs will not be able to buy Executive Select with their starting cash.

> Made cassetes and Souto from vendors more expensive. Buying food from Hot Foods now costs money. Marine coffee now has an appropiate description. Souto vendors no longer vend water bottles.

It's funny to make the lives of marines miserable.

> Fixed default parent type dollar items being worth 0 money...

This will let marines money scrounge.

# Testing Photographs and Procedure

<details>
<summary>Screenshots & Videos</summary>

Put screenshots and videos here with an empty line between the screenshots and the `<details>` tags.

Irrelevant.

</details>

# Changelog

:cl:
add: Added pay_multiplier to all paygrades. The higher your rank, the more money you'll start with, based on this multiplier. (For example, a Major will have a pay multiplier of 4.) Includes strange roles like VAIPO, UPP, PMCs, RESS...
del: Drastically nuked the amount of money people start with. People can no longer start with thousands of dollars.... they now get 30-50 dollars total. This value is multiplied by the pay_multiplier above.
spellcheck: Non-binary WY executives may now spawn with 'Mx.' as their communications prefix.
balance: Altered the prices of cigarette vending machines around to overall make them more expensive. PFCs will not be able to buy Executive Select with their starting cash.
del: Made cassetes and Souto from vendors more expensive. Buying food from Hot Foods now costs money. Marine coffee now has an appropiate description. Souto vendors no longer vend water bottles.
fix: Fixed default parent type dollar items being worth 0 money...
/:cl:

### CODE REVIEW

**Comment by ThePiachu** [comment]
> bal: Altered the prices of cigarette vending machines around to overall make them more expensive. PFCs will not be able to buy Executive Select with their starting cash.

It's `balance`, not `bal`

File: code/game/jobs/job/job.dm [hunk 1]
```diff
@@ -148,6 +148,31 @@
 /datum/job/proc/spawn_and_equip(var/mob/new_player/player)
 	CRASH("A job without a set spawn_and_equip proc has handle_spawn_and_equip set to TRUE!")
 
+/datum/job/proc/generate_money_account(mob/living/carbon/human/account_user)
+	account_user.job = title //TODO Why is this a mob variable at all?
```
Comment (fira) [inline code/game/jobs/job/job.dm:152]: Doesn't look like this should be in generate_money_account

**Review by fira — APPROVED** [review]
Tough times in the USCM

Code LGTM, remember to donate all the reclaimed budget to Dev Cabal

File: code/game/jobs/job/job.dm (hunk 1 above)
Comment (carlarctg) [reply code/game/jobs/job/job.dm:152]: guh

**Review by harryob — APPROVED** [review]
gamebreaking without game architect review :/ can't imagine what this would do to the balance

**Comment by stanalbatross** [comment]
dare I say it... soul?

**Review by fira — APPROVED** [review]

**Comment by Pyrodes** [comment]
"Non-binary WY executives may now spawn with 'Mx.' as their communications prefix."

why is this in a money rework and not it's own thing

**Comment by monkeysfist101** [comment]
> "Non-binary WY executives may now spawn with 'Mx.' as their communications prefix."
> 
> why is this in a money rework and not it's own thing

Because it’s a three character change to existing code in the money files that doesn’t actually do anything in game

**Pull request merged**